compiler-core/src/language_server/code_action.rs

```diff
@@ -1141,15 +1141,35 @@ impl<'a> QualifiedToUnqualifiedImportFirstPass<'a> {
             qualified_constructor: None,
         }
     }
-    fn get_module_import(&mut self, module_name: &EcoString) -> Option<&'a ast::Import<EcoString>> {
-        self.module
-            .ast
-            .definitions
-            .iter()
-            .find_map(|def| match def {
-                ast::Definition::Import(import) if import.module == *module_name => Some(import),
-                _ => None,
-            })
+    fn get_module_import(
+        &self,
+        module_name: &EcoString,
+        constructor: &EcoString,
+        is_type: bool,
```

Comment:
Tiny thing, rather than using a bool can we the the `Layer` type here. It has `Type` and  `Value` variants

Reply:
If you're up for making the larger change that would be very cool!